src/node_lkl.cc

```diff
@@ -13,11 +14,13 @@ NAN_METHOD(startKernel) {
 
 	lkl_host_ops.print = NULL;
 	int memory = info[0]->Uint32Value();
+	init_async();
```

Comment:
Can you add a FIXME comment about guarding the start and halt calls for double calls? Also we are deallocating the async handle that is referenced by the callbacks passed to the user of the lib. We should probably check that async is initialized in `callback_wrapper`